<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uxROIWidget</class>
 <widget class="QWidget" name="uxROIWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupROI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>ROI</string>
   </property>
   <widget class="QSpinBox" name="spinY1">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>60</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinX1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinX0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>22</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinY0">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>43</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGetROI">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>60</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
